contributor-guide/modules/ROOT/pages/testing/boost-test-matrix.adoc

```diff
@@ -1,286 +1,178 @@
-  --force-update        do an SVN update (if applicable) instead of a clean
-                        checkout, even when performing a full run
-  --have-source         do neither a tarball download nor an SVN update; used
-                        primarily for testing script changes
-  --ftp=FTP             FTP URL to upload results to.
-  --proxy=PROXY         HTTP proxy server address and port
-                        (e.g.'http://www.someproxy.com:3128')
-  --ftp-proxy=FTP\_PROXY
-                        FTP proxy server (e.g. 'ftpproxy')
-  --dart-server=DART\_SERVER
-                        the dart server to send results to
-  --debug-level=DEBUG\_LEVEL
-                        debugging level; controls the amount of debugging
-                        output printed
-  --send-bjam-log       send full bjam log of the regression run
-  --mail=MAIL           email address to send run notification to
-  --smtp-login=SMTP\_LOGIN
-                        STMP server address/login information, in the
-                        following form: <user>:<password>@<host>[:<port>]
-  --skip-tests          do not run bjam; used for testing script changes
-
-```
-
-To test develop use "`--tag=develop`",
- and to test master use
- "`--tag=master`". Or substitute any Boost
- tree of your choice.
-
-
-### Details
-
-
-The regression run procedure will:
-
-
-* Download the most recent regression scripts.
-* Download the designated testing tool sources including
- Boost.Jam, B2, and the various regression
- programs.
-* Download the most recent from the [Boost Git Repository](/users/download/#repository)
- into the subdirectory boost.
-* Build b2 and process\_jam\_log if
- needed. (process\_jam\_log is a utility, which
- extracts the test results from the log file produced by B2).
-* Run regression tests, process and collect the
- results.
-* Upload the results to a common FTP server.
-
-
-The report merger process running continuously will merge
- all submitted test runs and publish them at [various locations](testing.html#RegressionTesting).
-
-
-### Advanced use
+* Git (a recent version).
+* At least 5 gigabytes of free disk space per compiler to be tested.
+
+== Run Regression Tests Locally
+
+There are a range of options for running regression tests:
+
+. To run all of a library's regression tests, run the https://www.bfgroup.xyz/b2/[B2] utility from the `<boost-root>/libs/<library>/test` directory. 
+
+. To run a single test, specify the test name (found in `<boost-root>/libs/<library>/test/Jamfile.v2`) on the https://www.bfgroup.xyz/b2/[B2] command line.
+
+. To run every library's regression tests, run B2 from the `<boost-root>/status` directory.
+
+. To run B2's regression tests, run `python test\_all.py` from the `<boost-root>/tools/build/test` directory. 
+
+== Running Boost's Automated Regression and Reporting
+
+To run all Boost regression tests and report the results back to the Boost community:
```

Comment:
This section should explain what kind of tests these are. If it's about the "Boost Test Matrix", we should explain what that is, that people can submit their results, etc....